### PR TITLE
more events

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+  - date: 2015-02-20
```
Comment (dclim) [inline _data/events.yml]: wrong year

File: _data/events.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+    name: Druid Beijing Meetup
+    info: "Druid Big Data Architecture Technology Exchange"
+    link: http://www.huodongxing.com/event/4322200984100
+
   - date: 2015-03-30
```
Comment (dclim) [inline _data/events.yml]: wrong year

Comment (fjy) [inline _data/events.yml]: nice catch

**Comment by dclim** [comment]
:+1:

**Pull request merged**